lib/electric_slide/call_queue.rb

```diff
@@ -182,6 +176,28 @@ def add_agent(agent)
       async.check_for_connections
     end
 
+    # Updates a queued agent's attributes
+    def update_agent(agent, agent_attrs)
+      abort ArgumentError.new('Agent must be present') unless agent
```

Comment:
This log message is misleading. It should probably say `Agent must not be nil`. The current message makes it sound like the agent is not in the queue